src/accelerate/hooks.py

```diff
@@ -230,12 +233,12 @@ def __init__(
     ):
         self.execution_device = execution_device
         self.offload = offload
+        self.cpu_offload = cpu_offload
```

Comment:
Would it make sense to pass both `offload` and `cpu_offload`? It seems the former would take precedent over the latter. Maybe this could be checked or documented?